sdk/lib/opentelemetry/sdk/trace/span.rb

```diff
@@ -317,7 +317,7 @@ def initialize(context, parent_context, parent_span, name, kind, parent_span_id,
           # SpanData.
           @monotonic_start_timestamp = monotonic_now
           @realtime_start_timestamp = if parent_span.recording?
-                                        relative_realtime(parent_span.realtime_start_timestamp, parent_span.monotonic_start_timestamp)
+                                        relative_realtime(parent_span.realtime_start_timestamp, parent_span.monotonic_start_timestamp, @monotonic_start_timestamp)
```

Comment:
Here, every `Span#initialize` was calling `Process.clock_gettime` twice: once to initialize `@monotonic_start_timestamp`, and another time in the implementation of `relative_realtime`. I noticed that, by passing a time _in_ to that method, we might save that second call out to the system function. 

Another use of `relative_realtime` had to be updated below, it calls `monotonic_now` directly and passes the result.